src/dashboard/user-interface/general-page-integration.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Yoast\WP\SEO\Dashboard\User_Interface;
+
+use WPSEO_Admin_Asset_Manager;
+use Yoast\WP\SEO\Conditionals\Admin_Conditional;
+use Yoast\WP\SEO\Conditionals\New_Dashboard_Ui_Conditional;
+use Yoast\WP\SEO\Helpers\Current_Page_Helper;
+use Yoast\WP\SEO\Helpers\Product_Helper;
+use Yoast\WP\SEO\Helpers\Short_Link_Helper;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+
+/**
+ * Class General_Page_Integration.
+ */
+class General_Page_Integration implements Integration_Interface {
+
+	/**
+	 * The page name.
+	 *
+	 * @TODO RENAME THIS TO SOMETHING SENSIBLE AFTER FEATURE FLAG PERIOD
+	 */
+	public const PAGE = 'wpseo_page_dashboard_new';
```

Comment:
In order for us to not miss anything when the time comes to rename all this to the final name, I'd suggest if we kept the same temp naming everywhere. Currently we have `dashboard` (in the packages directory, in the submenu identifier, in the JS initializer, etc. ), `dashboard new` (here, in the page title, etc), `new dashboard` (in the feature flag naming, in the script/sheet naming, etc.), or even `general` (in the name of the integration).

Let's keep this unified so we can later rename everything to the final form easily.

Reply:
I have used the new_dashboard name where applicable. I have kept the dashboard name in the packages so its easier to keep working in that directory is we want to use it for upgrading the dashboard to the actual dashboard.